docs/src/cloud.md

```diff
@@ -0,0 +1,51 @@
+# Cloud Execution
+
+## Azure
+
+A key benefit and design motivation of OpticSim is being able to execute many simulations/optimizations at once.
+This is best enabled through the use of cloud computing services such as Azure.
+
+As part of the base package, we provide support for cloud execution using an [Azure Machine Learning](https://azure.microsoft.com/en-gb/free/machine-learning) workspace.
+
+To use this functionally you'll first need to set up an AML workspace with a compute cluster. Then you'll need to provide a few bits of information to OpticSim:
+
+- Subscription ID, of the form `XXXXXXXX-XXX-XXX-XXXX-XXXXXXXXXXXX`
+- Resource group name
+- Workspace name
+- Compute cluster name
+
+This information can be cached either to a specific file, or globally:
+
+```@docs
+Optics.Cloud.cache_run_config
```

Comment:
Need to change Optics -> OpticSim

Reply:
Yeah, publishing works great - definitely a useful feature https://microsoft.github.io/OpticSim.jl/previews/PR71/cloud/